steward/src/cellars/aave_v2_stablecoin.rs

```diff
@@ -25,6 +25,13 @@ const CELLAR_NAME: &str = "aave_v2_stablecoin";
 const LOG_PREFIX: &str = "AaveV2StablcoinCellar";
 
 pub fn get_encoded_call(function: StrategyFunction, cellar_id: String) -> Result<Vec<u8>, Error> {
+    get_call(function, cellar_id).map(|call| call.encode())
```

Comment:
Yes, this return `Vec<u8>` (not `Vec<Bytes>`) which is a single call

Reply:
Right, but `map` is applied to elements of an iterator. Unless `get_call` is returning an array of calls to encode, isn't this going to be calling the `map` function on every element of the incoming `vec`? I'm confused as to why this would be working.

Reply:
What happens when `map` is called on a `Result` type here without checking to see if it's `Error`?